readme.md

```diff
@@ -106,7 +125,25 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+//CREATE
+app.post("/cats", function(req, res){
+  console.log("create")
+});
+
+//READ
+app.get("/cats", function(req, res){
+  console.log("index")
+});
+
+//UPDATE
+app.put("/cats/:cat_id", function(req, res){
+  console.log("update")
+});
+
+// Destroy
+app.delete("/cats/:cat_id", function(req, res){
+  console.log("delete")
+})
 
```

Comment:
The only problem with using `console.log` is that it will hang. Because a `res` is never sent back from the request. Switch all `console.log`'s to `res.send()`